libs/trustchain/src/sdk.ts

```diff
@@ -203,6 +206,9 @@ export class SDK implements TrustchainSDK {
       memberCredentials.pubkey !== member.id,
       "removeMember must not be used to remove the current member.",
     );
+
+    const hook = await this.lifecycle?.onTrustchainRotation(this, trustchain, memberCredentials);
```

Comment:
can we find a better naming? 
 `rotation` will be more suitable I think

I mean because you use a `if` and by definition Hooks can be used like that

Reply:
ok i will probably rename to `afterRotation` then 👍